packages/ag-charts-community/src/chart/series/series.ts

```diff
@@ -480,6 +480,7 @@ export abstract class Series<
 
     destroy(): void {
         this.destroyFns.forEach((f) => f());
+        this.destroyFns.length = 0;
```

Comment:
```suggestion
        this.destroyFns = [];
```
nit: this is a hacky way to reset an array, I think we should avoid doing it.